src/lib/geolocationQuota/lib.js

```diff
@@ -72,5 +93,19 @@ export const checkAndSendGeolocationQuotaNotification = async (
       notificationData
     )
     await sendNotification(client, notificationView)
+  } else if (remainingDays === 3) {
```

Comment:
`3` en dur ? Pas de gestion par flag ou autre ?

Reply:
It was not planned but I will check to see if it is a miss. Anyway I moved the value up in a const to make it quickly readable.